refarch/aws-native/cdk.context.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I considered putting this in the `.example` file and gitignore this file (to avoid having contributors accidentally put their accounts here)

But end users need to commit these information in the repo anyway (as Pipeline will deploy the latest version from code repo, not local) so I leave it as it is.

Reply:
yeah and the issue here is that you will get value in your tryGetContext with `<CICD_ACCOUNT>` string value and fail late in the process ... agree on pushing to example and rely on throw exception to remind customer